mews-operations/availabilityBlock.md

```diff
@@ -217,8 +227,15 @@
 | `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2024-12-05"`\). |
 | `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2024-12-31"`\). |
 | `prices` | [`Price`](../channel-manager-operations/inventory.md#price) collection | required | Collection of prices for each guest count. |
+| `occupancyAdjustments` | [`OccupancyAdjustment`](#occupancy-adjustment) collection | optional | Collection of extra occupancy adjustments for various age categories. |
```

Comment:
Strictly I think it should be "array of..." rather than "collection" because "collection" is not an official JSON type, but I see the type "collection" has already been used in availability blocks. Either way it will be clear to a developer, it just depends how strict we want to be.

Reply:
Unified in the block file